### ISSUE TITLE
Moderation team recertification

### ISSUE THREAD

**Issue by Trott** [nodejs/admin#408]
The last time @nodejs/community-committee and @nodejs/tsc certified members of @nodejs/moderation was August-October of 2018. (See https://github.com/nodejs/community-committee/issues/361 for CommComm.)

They need to be certified annually. So, I'm adding this to the agenda for both TSC and CommComm.

In addition to certifying (or not certifying, and thus removing) members of the Moderation Team, it might also be a good time to either think about people we might ask to serve and/or put a call out for volunteers.

**Comment by mhdawson** [comment]
I think it's also a good time to thank the moderation team members for all the hard work over the last year.  I think the've been doing a great job.

**Comment by mcollina** [comment]
Great job folks!

**Comment by Fishrock123** [comment]
No objections to recertifying here.

**Comment by jasnell** [comment]
No objections. +1 to recertifying.

**Comment by Trott** [comment]
TL;DR: Hey, @bnb, @dshaw, @gdams, @gibfahn, @hackygolucky, @thefourtheye: can you folks indicate whether you are active in Moderation these days and wish to remain on the team? Whatever your answer, thank you for doing the hard Moderation Team work this year!

Current Moderation Team members are:

* @benjamingr
* @bnb
* @dshaw
* @gdams
* @gibfahn
* @hackygolucky
* @ljharb
* @othiym23
* @ryanmurakami
* @thefourtheye

Of those team members, @benjamingr and @ryanmurakami are active based on activity in the moderation repository.@othiym23 [chimed in on behalf of Moderation Team](https://github.com/nodejs/TSC/issues/744#issuecomment-521384866) not that long ago. @ljharb [affirmed their activity/engagement too](https://github.com/nodejs/TSC/issues/752#issuecomment-527222898). 

AFAICT, Moderation Team has not had a public meeting for a long time and I don't know where their minutes are kept. I suspect most of the conversations happen in the private Slack channel. So, rather than try to dig through a public record that I don't know where to find, I'm just flat-out asking folks if they've been doing stuff.

**Comment by othiym23** [comment]
I am interested in continuing to serve on the moderation team.

**Comment by bnb** [comment]
I am interested in continuing to serve on the moderation team as well ❤️

**Comment by bnb** [comment]
On activity: we’re in an interesting place where reactions happen very rapidly and are taken care of by the time I get to them. As such, I don’t engage as I’ve found dogpiling is not an effective strategy for moderation 😅

**Comment by dshaw** [comment]
I am unable to allocate the time needed to serve on this team and will not be continuing this year. Thank you. 💖✨

**Comment by hackygolucky** [comment]
Good question, and it is time for me to step down. 
I have too many other priorities to focus on for noow, and my work has def pulled away from this work. Certainly not doing any favors here by taking up a seat. 🤗

**Comment by mhdawson** [comment]
We discussed in the TSC meeting today the following proposal from Rich:

    * Re-certify the following people:
      * @benjamingr
      * @bnb
      * @ljharb
      * @othiym23
      * @ryanmurakami
  * We should not re-certify these people as they have voluntarily removed themselves from the team
    or have not responded, but can always be added at a later time if they decide they want to be
    part of the team:
      * @dshaw
      * @hackygolucky
      * @gdams
      * @gibfahn
      * @thefourtheye

And the following people in attendance will be counted as voting +1

* Anna Henningsen @addaleax (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC) (Partially)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Sam Roberts @sam-github (TSC)
* Tobias Nießen @tniessen (TSC)


@nodejs/tsc if you are able to the meeting, please add your vote to approve or reject the re certification as proposed.

**Comment by targos** [comment]
+1

**Comment by Trott** [comment]
+1 as well. With my vote, that brings us to 11 TSC members out of 20 voting to recertify these people:

* @benjamingr
* @bnb
* @ljharb
* @othiym23
* @ryanmurakami

All members of Moderation Team must be re-certified by both CommComm and TSC. Since those are the only individuals being re-certified by TSC, they are also the only individuals that CommComm needs to consider at this time.

@nodejs/community-committee: If you could confirm certification for those five people, we can close this out. (A subsequent issue to nominate/recruit new members would be good, but can be a separate thing.)

**Comment by bnb** [comment]
@Trott I will abstain, but will bring this up in today's CommComm meeting.

**Comment by benjamingr** [comment]
@Trott first of all thank you for doing this 🙇 I am concerned that this means there is no TSC representation in the moderation team which I think is important for establishing trust between the moderation team and various bodies.

Would it be possible to ask for volunteers in the next TSC meeting?

**Comment by Trott** [comment]
> Would it be possible to ask for volunteers in the next TSC meeting?

@benjamingr Yes. Also, I volunteer, but will wait until this first round of re-certification is done to put my hand up.

**Comment by mhdawson** [comment]
In the commcomm meeting meeting today the following commcomm members voted to recertify based on the recommendation:

* Michael Dawson (@mhdawson)
* Manil Chowdhury (@keywordnew)
* Tierney Cyren (@bnb)
* Adam Miller (@Trott amiller-gh)
* Joe Sepi (@joesepi)
* Waleed Ashraf (@waleedashraf)

@nodejs/community-committee could those of you who could not make the meeting please cast your vote in this issue or contact me with your concerns if you don't feel comfortable adding them to the issue.

**Comment by Trott** [comment]
> In the commcomm meeting meeting today the following commcomm members voted to recertify based on the recommendation:
> 
> * Michael Dawson (@mhdawson)
> * Manil Chowdhury (@keywordnew)
> * Tierney Cyren (@bnb)
> * Adam Miller (@amiller-gh)
> * Joe Sepi (@joesepi)
> * Waleed Ashraf (@WaleedAshraf)
> 
> @nodejs/community-committee could those of you who could not make the meeting please cast your vote in this issue or contact me with your concerns if you don't feel comfortable adding them to the issue.

Aside, but a relevant and important one: @mhdawson https://github.com/nodejs/community-committee lists 13 CommComm members but @nodejs/community-committee has 14 members. Which (if either) is correct? The README lists @AhmadAwais but they're not on the GitHub team. Meanwhile, the GitHub team includes @jemjam and @Tiriel, but they are listed as Emeriti in the README. So I'd guess that the README is correct and the team needs to be updated?

**Comment by mcollina** [comment]
+1 from me

**Comment by MylesBorins** [comment]
+1 from me

**Comment by ahmadawais** [comment]
🙌

+1 from me as well for the moderation team recertification. Mad props on
all the work you folks do.

Stuck at the US Embassy for a couple of days, just returning back so
couldn't make it to the meeting but watched it on YouTube just now.

>The README lists @ahmadawais <https://github.com/ahmadawais> but they're
not on the GitHub team

This has come before, I am not sure how to fix this. Is there anything I
need to do? Otherwise, please add me there to sort this through.

Looking forward, peace! ✌️



AHMAD AWAIS <https://twitter.com/MrAhmadAwais/>
❯ Open Source & JavaScript Dev Advocate ❯ Node.js Foundation Community
Committee Member
❯ Google Developers Expert for Web Tech ❯ WordPress Core Developer ❯ EE-CS
Engineer
❯ Hilarious comedian ❯ Self-confessed technology addict. ;)

*→ Read my blog: AhmadAwais.com <https://ahmadawais.com/> *
*→ *

*Twitter <https://twitter.com/MrAhmadAwais/> | GitHub
<https://github.com/AhmadAwais> | Facebook
<https://www.facebook.com/ahmadawais> | LinkedIn
<https://www.linkedin.com/in/MrAhmadAwais/> → Skype/Slack &
More: MrAhmadAwais → I've released a course on VSCode.pro
<https://vscode.pro/?utm_source=EmailFooter>  *

**Comment by bnb** [comment]
@ahmadawais / @Trott: Fixed.

**Comment by mhdawson** [comment]
@bnb thanks, I was going to hope you answered Rich's question/concerns.

**Comment by Trott** [comment]
With this, it appears that the five members are recertified. I"m going to close this. If I'm wrong to do that because CommComm's procedure is different than I imagine, please re-open.

**Comment by Trott** [comment]
P.S. @nodejs/moderation I've taken care of all the documented Moderation Team offboarding tasks except for removing the offboarded folks from the Moderation Team Slack because I do not have access to that Slack. Please do that! Thanks!

**Comment by benjamingr** [comment]
@Trott awesome super helpful. I have pinged the Ryan in the slack. And you know... if you ever feel like joining again 😉

**Comment by DennisRichi** [comment]
Another benefit of [blackout curtains](https://www.superiorbrand.com/collections/blackout-curtains) is that they can help control room temperature. The thick fabric creates an extra layer between the window and the room. During hot weather, they can reduce the amount of heat coming through the windows. In colder months, they can also help reduce heat loss and keep the room more comfortable.
